remote/rtRemoteServer.cpp

```diff
@@ -579,6 +579,46 @@ rtRemoteServer::findObject(std::string const& objectId, rtObjectRef& obj, uint32
 }
 
 rtError
+rtRemoteServer::unregisterDisconnectedCallback( clientDisconnectedCallback cb, void *cbdata )
```

Comment:
I've just realized that we don't need the callback pointer to do the job, just a data pointer is required. Should I fix this?